dallinger/command_line.py

```diff
@@ -363,7 +363,7 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1):
     try:
         team = config.get("Heroku Access", "team")
         log("Trasferring to {} team...".format(team))
-        subprocess.call([
+        subprocess.check_call([
```

Comment:
Just to double check — because this is in a try/except, it will continue even if the subprocess call fails, right?